app/config.js

```diff
@@ -41,4 +41,4 @@ window.OPENSHIFT_VERSION = {
   kubernetes: "dev-mode"
 };
 
-window.OPENSHIFT_CONSOLE_BASE_URL = "https://localhost:9000/dev-console";
+window.OPENSHIFT_CONSOLE_BASE_URL = "https://localhost:" + (window.DEV_SERVER_PORT || '9001');
```

Comment:
> The sometimes working take you out of the showcase application.

But that's what's going to happen when the catalog is running outside of the console. Why not make the showcase app work like it will really work? And then we can test the base URL customization.

I'm OK building the URL form the API server config, but I'd rather not change it to something we know is broken.